<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件标签化管理工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="twFileBrowser">
      <widget class="QWidget" name="twFileBrowserPage1">
       <attribute name="title">
        <string>文件查看</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="FileBrowser" name="wFileBrowser" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFileManage">
    <property name="title">
     <string>文件管理</string>
    </property>
    <addaction name="actionAddLabels"/>
    <addaction name="actionDelLabels"/>
   </widget>
   <widget class="QMenu" name="menuLabelManage">
    <property name="title">
     <string>标签管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFileSearchCondition">
    <property name="title">
     <string>查找条件</string>
    </property>
    <addaction name="actionFromCache"/>
    <addaction name="actionFromDisk"/>
    <addaction name="separator"/>
    <addaction name="actionLabelLogicAnd"/>
    <addaction name="actionLabelLogicOr"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuFileSearch">
    <property name="title">
     <string>查找文件</string>
    </property>
    <addaction name="actionSearchFilesbyLabels"/>
    <addaction name="actionTraverseSelDirs"/>
   </widget>
   <addaction name="menuFileManage"/>
   <addaction name="menuLabelManage"/>
   <addaction name="menuFileSearch"/>
   <addaction name="menuFileSearchCondition"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddLabels"/>
   <addaction name="actionDelLabels"/>
   <addaction name="actionSearchFilesbyLabels"/>
   <addaction name="actionTraverseSelDirs"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dwLabels">
   <property name="windowTitle">
    <string>标签库</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionSearchFilesbyLabels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/search.png</normaloff>:/Resources/search.png</iconset>
   </property>
   <property name="text">
    <string>根据标签查找文件</string>
   </property>
   <property name="toolTip">
    <string>根据标签查找文件</string>
   </property>
  </action>
  <action name="actionAddLabels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/204-link.png</normaloff>:/Resources/204-link.png</iconset>
   </property>
   <property name="text">
    <string>为文件关联标签</string>
   </property>
   <property name="toolTip">
    <string>为文件关联标签</string>
   </property>
  </action>
  <action name="actionTraverseSelDirs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/database.png</normaloff>:/Resources/database.png</iconset>
   </property>
   <property name="text">
    <string>遍历选中的文件夹</string>
   </property>
   <property name="toolTip">
    <string>遍历选中的文件夹</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/question.png</normaloff>:/Resources/question.png</iconset>
   </property>
   <property name="text">
    <string>界面介绍</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionFromCache">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>优先从缓存数据库查找</string>
   </property>
  </action>
  <action name="actionLabelLogicAnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>标签的逻辑关系为与</string>
   </property>
  </action>
  <action name="actionDelLabels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/breaklink.png</normaloff>:/Resources/breaklink.png</iconset>
   </property>
   <property name="text">
    <string>删除标签</string>
   </property>
  </action>
  <action name="actionFromDisk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>直接从磁盘查找</string>
   </property>
  </action>
  <action name="actionLabelLogicOr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>标签的逻辑关系为或</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>作者信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileBrowser</class>
   <extends>QWidget</extends>
   <header>filebrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
